### DISCUSSION TITLE
The so_vits_svc_fork interface doesn't export an audio file when 'Infer' is pressed

### DISCUSSION THREAD

**Discussion by mariusmilatz** [voicepaw/so-vits-svc-fork#382] [Q&A]
I've installed the so_vits_svc_fork on my computer. I've both tried my own model and a model from the discord community. both time when i press the 'Infer' buttorn want watch the tasks being executed, it stops at 'Finish inference for audio 1.wav'. I've waited half a hour to see if anything changes but nothing happens. does anyone have a solution to this problem?

![Screenshot (915)](https://user-images.githubusercontent.com/41435955/232731303-4ed58a75-76d6-40ec-ad41-78601f4a59cc.png)
![Screenshot (916)](https://user-images.githubusercontent.com/41435955/232731313-be5604d3-dad1-4a12-8ee0-62096656278c.png)

**Comment by Air1N** [comment]
It should play the inferred audio out loud and create an audio file `name of original file.out.wav` in the same folder as the original audio file. I am not having this issue. Could we get more details? It is only doing one chunk so perhaps you have your "silence threshold" too high, or maybe the file is not long enough? Mine does several of the `Chunk: Chunk{Speech: ...`

Another question, why is the folder named `NewFolder/drake05` but you're using `speaker: drake01` ?

**Reply by mariusmilatz** [reply]
thanks for trying to help me first. i appreciate it. the folder name is the model from the discord community. i have changed it so the folder is 'drake01' but i get the same error. i have also used my own model which has a 'silence threshold' of 5. on the tutorial i used to create the model they said use maximum of 25. if 5 to high? the audio file that i want changed is 3min long. my computer has Nvidia Geforce RTX 3060ti and a AMD Ryzen 5 3600 6-Core Processor.

**Reply by Air1N** [reply]
> i have changed it so the folder is 'drake01' but i get the same error.

I intended for you to check if there are any other speakers in the dropdown in svcgui. Sorry if none of this helped, otherwise I have no idea. Only lead is that it only did one chunk, whereas mine does several chunks like this (most recent version): 
![image](https://user-images.githubusercontent.com/12143899/233187993-79155268-9bdd-497c-ac1a-122665affcca.png)

**Reply by mariusmilatz** [reply]
thanks for trying to help me. i appreciate it. there was on other speakers in the drop down menu. ill try and uninstall the script and reinstall it. hopeful that will work. thanks again.

**Comment by sbersier** [comment]
@mariusmilatz  : Have you solved your problem? If not, try to update  so_vits_svc_fork.
